dtfs-central-api/src/cron-scheduler-jobs/deal-cancellation/cancel-deal-job.ts

```diff
@@ -0,0 +1,39 @@
+import { asString, CronSchedulerJob, TfmDeal } from '@ukef/dtfs2-common';
+import { generateSystemAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { endOfDay } from 'date-fns';
+import { TfmDealCancellationRepo } from '../../repositories/tfm-deals-repo';
+
+const { DEAL_CANCELLATION_SCHEDULE } = process.env;
+
+/**
+ * Updates deals to be cancelled
+ * @param dealIds the deals to be cancelled
+ */
+const cancelDeals = async (deals: TfmDeal[]) => {
+  await Promise.all(deals.map((deal) => TfmDealCancellationRepo.submitDealCancellation(deal._id, deal.tfm.cancellation!, generateSystemAuditDetails())));
+};
+
+/**
+ * Gets deal ids with a scheduled cancellation & effective date today or in the past
+ * @returns The deal ids to cancel
+ */
+const getDealsWithCancellationsScheduledForThePast = async (): Promise<TfmDeal[]> => {
+  const dealsScheduledForCancellation = await TfmDealCancellationRepo.findScheduledDealCancellations();
+
+  return dealsScheduledForCancellation.filter((deals) => deals.tfm.cancellation!.effectiveFrom < endOfDay(new Date()).valueOf());
+};
+
+/**
+ * Cancels deals scheduled to be cancelled in the past
+ */
+const cancelScheduledDeals = async (): Promise<void> => {
+  const deals = await getDealsWithCancellationsScheduledForThePast();
```

Comment:
```suggestion
  const deals = await getDealsWithPastCancellationsScheduled();
```

In the interest of keeping names short.